presto-spi/src/main/java/com/facebook/presto/spi/type/Decimals.java

```diff
@@ -232,6 +234,14 @@ public static void checkOverflow(BigInteger value)
         }
     }
 
+    public static BigDecimal readBigDecimal(DecimalType type, Block block, int position)
```

Comment:
"getBigDecimal" ?

Reply:
I called it "read" as it's the inverse of the existing "write" method. Either one seems fine here.